collector.go

```diff
@@ -0,0 +1,59 @@
+package versionbundle
+
+import (
+	"net/url"
+
+	"github.com/giantswarm/microerror"
+	"github.com/go-resty/resty"
+)
+
+type CollectorConfig struct {
+	RestClient *resty.Client
+
+	Endpoints []url.URL
+}
+
+func DefaultCollectorConfig() CollectorConfig {
+	return CollectorConfig{
+		RestClient: nil,
+
+		Endpoints: nil,
+	}
+}
+
+type Collector struct {
+	restClient *resty.Client
+
+	bundles []Bundle
+
+	endpoints []url.URL
+}
+
+func NewCollector(config CollectorConfig) (*Collector, error) {
```

Comment:
We have to collect version bundles from all the microservices and operators that expose any. Therefore we need to request all version endpoints that are of interest for us. The idea here is to have a collector to do this job.